examples/src/bin/css.rs

```diff
@@ -0,0 +1,97 @@
+//! # CSS example
+//!
+//! This sample demonstrates how to use CSS with gtk-rs.
+
+use gio::prelude::*;
+use gtk::prelude::*;
+
+use std::env::args;
+
+use gdk::Display;
+use gtk::{
+    Application, ApplicationWindow, Box as Box_, Button, ComboBoxText, CssProvider, Entry,
+    Orientation, StyleContext, STYLE_PROVIDER_PRIORITY_APPLICATION,
+};
+// Basic CSS: we change background color, we set font color to black and we set it as bold.
+const STYLE: &str = "
+entry.entry1 {
+    background: linear-gradient(to right, #f00, #0f0);
+    color: blue;
+    font-weight: bold;
+}
+
+button {
+    /* If we don't put it, the yellow background won't be visible */
+    background-image: none;
+}
+
+button.button1:hover {
+    transition: 500ms;
+    color: red;
+    background-color: yellow;
+}
+
+combobox button.combo box {
+    padding: 5px;
+}
+combobox box arrow {
+    -gtk-icon-source: -gtk-icontheme('pan-down-symbolic');
```

Comment:
Is this still possible with gtk4?

Reply:
I tried -gtk-icon-source: -gtk-icontheme('pan-up-symbolic'); (UP) and it seemed to work ok.